cli/bin/kontena

```diff
@@ -13,6 +13,6 @@ if ARGV[0] == 'complete'
   require 'kontena/scripts/completer'
 else
   require 'kontena_cli'
-  Kontena::PluginManager.instance.init
+  Kontena::PluginManager.instance.init unless ARGV.include?('plugin')
```

Comment:
As long as someone doesn't try and `kontena digitalocean master create --name plugin` :crab: